src/metabase/integrations/google.clj

```diff
@@ -115,12 +115,23 @@
   ;; things hairy and only enforce those for non-Google Auth users
   (user/create-new-google-auth-user! new-user))
 
+(defn- update-google-user!
```

Comment:
nit: when a function only does something conditionally, we often prefix it with `maybe-` to make that clear
```suggestion
(defn- maybe-update-google-user!
```